frontend/src/containers/Homepage/Homepage.js

```diff
@@ -40,9 +41,13 @@ const Homepage = ({ location, user, setJwt }) => {
           <div className="row">
             <RecipeSearchList searchTerm={searchTerm} />
           </div>
-          <div className="row">
-            <BottomSignUp />
-          </div>
+          { isNil(user) ? (
+            <div className="row">
+              <BottomSignUp />
+            </div>
+          )
+            : null
```

Comment:
The spacing/formatting is a little off here. I think something like this would be a little cleaner:
```
{ isNil(user) ? (
    <div className="row">
      <BottomSignUp />
    </div>
  ) : null
}